### Investment Account Option Problem 

Option 1: Compounded Annually 

Account A Details:

- r = 0.08
- t = 8 years 
- n = 1 
- P = $ 50,000 

Formula:

$$ A(8) = 50,000(1 + \frac{0.08}{1})^{8} $$ 

In [1]:
def accountA(P, n, r, t):
    interest = P*(1 + (r/n))**(n*t)
    return interest 

print(accountA(50000,1,0.08,8))

92546.51051409413


In [2]:
92546.51 - 50000

42546.509999999995

Interest_A = 92546.51 - 50,000 = 42546.51 

Option 2: Compounded Continuously 

Account B Details:

- r = 0.07
- t = 10 years 
- P = 50,000


Formula:

$$ A(10) = 50,000*e^{0.08*10} $$ 

In [3]:
import math as m
def accountB(P,r,t):
    interest = P*m.e**(r*t)
    return interest 

print(accountB(50000,0.07,10))

100687.63537352384


In [4]:
100687.63 - 50000

50687.630000000005

Interest_B = 50687.63 

Comparing the two interests, the best account to invest in is Account B as it provides the most interest. 